lib/health.js

```diff
@@ -1,70 +1,61 @@
 var os = require('os'),
     df = require('node-df'),
-    path = require('path');
+    path = require('path'),
+    pkgDir = require('pkg-dir');
 
-function loadMainPackageJSON(attempts) {
-  attempts = attempts || 1;
-    if (attempts > 5) {
-        throw new Error('Can\'t resolve main package.json file');
-    }
-    var mainPath = attempts === 1 ? './' : Array(attempts).join("../");
-    try {
-        return require.main.require(mainPath + 'package.json');
-    } catch (e) {
-        return loadMainPackageJSON(attempts + 1);
-    }
-}
-
-var pjson = loadMainPackageJSON();
 var DEFAULT_PATH = '/ping';
 
 /**
  * Get system informaton
  * @param {Function} cb
  */
 function info(cb) {
-  df(function (error, diskInfo) {
-    if (error) {
-      diskInfo = {msg: 'Error fetching disk info', error: error};
-    }
+  pkgDir(path.resolve(__dirname, '../')).then(function(x) {
```

Comment:
Still not sure about whether this works when you use express-ping as a module in a bigger project that contains its own package.json. Which package.json file would this code find? The one that belongs to express-ping or the one that belongs to the bigger project?